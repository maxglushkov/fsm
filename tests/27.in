R
2
S	| 0A
A
