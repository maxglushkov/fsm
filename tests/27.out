2
1
SX 0 AF
